src/tooltip/tooltip.spec.ts

```diff
@@ -28,15 +28,14 @@ describe('ngb-tooltip-window', () => {
     fixture.detectChanges();
 
     expect(fixture.nativeElement).toHaveCssClass('tooltip');
-    expect(fixture.nativeElement).toHaveCssClass('bs-tooltip-top');
+    expect(fixture.nativeElement).not.toHaveCssClass('bs-tooltip-top');
     expect(fixture.nativeElement.getAttribute('role')).toBe('tooltip');
   });
 
-  it('should position tooltips as requested', () => {
+  it(`shouldn't add 'bs-tooltip-left' class with binding`, () => {
```

Comment:
Why should it even try applying left if you removed `fixture.componentInstance.placement = 'left';` ?

this isn't testing anything now, you should put it back :)

See what you did in `popover.spec.ts`